tasks/request_files/test/unit_tests/test_request_files.py

```diff
@@ -108,6 +108,55 @@ def test_task_happy_path(self, mock_inner_task: MagicMock):
             db_queue_url,
         )
 
+    @patch("request_files.inner_task")
```

Comment:
May want a test to validate that default is taken if override not set.

Reply:
I treated the other tests as base cases, and this as testing the override.